includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -85,8 +85,8 @@ public static function register() {
 			self::POST_TYPE_SLUG,
 			array(
 				'labels'       => array(
-					'name'               => _x( 'Invalid AMP Pages (URLs)', 'post type general name', 'amp' ),
-					'menu_name'          => __( 'Invalid Pages', 'amp' ),
+					'name'               => _x( 'Errors by URL', 'post type general name', 'amp' ),
+					'menu_name'          => __( 'Errors by URL', 'amp' ),
```

Comment:
Let's change these to “Invalid URLs” throughout. As part of this, we can remove code related to bolding the “URLs” string.